.github/workflows/_charm-quality-checks.yaml

```diff
@@ -56,12 +56,21 @@ jobs:
         with:
           fetch-depth: 1
       - name: Check charm libraries # Make sure our charm libraries are updated
-        uses: canonical/charming-actions/check-libraries@2.5.0-rc
-        with:
-          credentials: "${{ secrets.CHARMHUB_TOKEN }}"
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
+        env:
+          CHARMHUB_TOKEN: "${{ secrets.CHARMHUB_TOKEN }}"
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           charm-path: "${{ inputs.charm-path }}"
           charmcraft-channel: "${{ inputs.charmcraft-channel }}"
+        run: |
+          pip install git+https://github.com/lucabello/noctua
```

Comment:
Can we add tags in this target repo and use them here?  That way we can make changes to the package without them always going live here

Reply:
Good point, I will add this :)

Reply:
Prior to merging this, can we have a CI set up, similar to what we have in cos-lib, so that noctua is auto tagged on release?